css-canvas-medical/about-us.html

```diff
@@ -0,0 +1,251 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <title>About Us</title>
+</head>
+<body>
+  <header class="margin-top under-line">
+    <div class="row space-between width">
+      <div class="row align-items-center">
+        <div class="img-container">
+          <img src="./images/logo-blue.svg" alt="Canvas">
+        </div>
+      </div>
+      <div class="column-full">
+        <nav class="nav row align-items-center">
+          <a href="product.html">product</a>
+          <a href="implementation.html">implementation</a>
+          <a href="about-us.html">about us</a>
+          <a href="#"> blog</a>
+          <button>Contact us</button>
+        </nav>
+      </div>
+    </div>
+  </header>
+  <div class="container">
+    <main>
+      <section>
+        <div class="row margin-top">
+          <div class="column-full position-relative">
+            <div class="overlay"></div>
+            <div class="hero-img-container">
+              <img src="./images/about-us/about-us-header.jpg" alt="about us header image">
+            </div>
+            <div class="text-container">
+              <h1>Clinicians should have software they love.</h1>
+              <p class="line-height">Our mission is to give clinicians and their teams highly usable,
+                  intelligent tools to advance their practice of medicine &mdash; and improve their day-to-day.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="margin-top-bottom">
+        <div class="text-align-center">
+          <h2>What We Believe</h2>
+        </div>
+        <div class="about-us-text">
+          <p>Why did we enter this crowded field? Because medical records have fallen
+            dramatically short of their promise to improve the quality and efficiency of health
+            care. Clinicians spend far too long at their computers. Workflows are overly restrictive.
+            Valuable medical data is left unstructured.
+          </p>
+          <p>We are honored, and humbled, when a practice chooses us as their technology partner. Medicine is a delicate
+            craft that mixes art and science &mdash; technology can help or hinder this craft. Working with
+            clinicians across the country, we think we have begun to build something that helps.
+          </p>
+          <a href="blog.html" class="link">check out our blog for updates, data, and essays on Canvas and American
+            primary care.
```

Comment:
The coloring and capitalization here is off:

---

![image](https://user-images.githubusercontent.com/16143771/152256600-69323660-d2ce-4555-8121-6c1e1ed5cc65.png)

---

![image](https://user-images.githubusercontent.com/16143771/152256620-ecc53e81-f740-44ea-a3c7-386dd238201d.png)